src/components/DynamicFormContainer/DynamicForm/CustomTemplates/CustomFileWidget.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { FunctionComponent, useCallback, useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { showInKB } from "../../../../common/utils";
+import { Button } from "../../../../UI/Button";
+import { SvgIcon, SvgIconPaperClip, SvgIconX } from "../../../../UI/SvgIcon";
+
+interface FileUploadType {
+  dataURL: string;
+  name: string;
+  size: number;
+  type: string;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const CustomFileWidget: FunctionComponent<any> = ({
+  onChange,
+  multiple,
+  options,
+  disabled,
+}) => {
+  const [uploadedFiles, setUploadedFiles] = useState<FileUploadType[]>([]);
+  const [fileSizeError, setFileSizeError] = useState(false);
+
+  const maxSize = 20000000;
+
+  const onDrop = useCallback(
+    (files: File[]) => {
+      let totalSize = uploadedFiles.reduce((acc, current) => acc + current.size, 0);
+      files.forEach((file: File) => {
+        totalSize += file.size;
+      });
+
```

Comment:
Can change to 

`if(totalSize > MAX_FILE_SIZE) return setFileSizeError(true);`

This will remove one layer of nesting below for better readabilty.

Reply:
ok got it, but i need that forEach fn to tabulate the file size of the files that was just uploaded.